tasks/service-wrapper/Dockerfile.asf-gdal

```diff
@@ -0,0 +1,17 @@
+ARG BASE_IMAGE=asfdataservices/gdal-subsetter
```

Comment:
I agree this is pretty brittle. 
- What if service provider changes their base image from ubuntu to alpine, would need to rewrite this image. 
- Could also run into version/dependency conflicts of installed packages, if SP also uses node but needs node6. 
- User permissions are also a problem, l2ss-py image has a `dockeruser` user that has permissions to the installed software, not sure that this `node` user would be permitted to run the python module. 
- How do you know if a python environment needs to be activated in order to run the service provider software? 
- Do you run snyk on all of these new images? Are you handling security updates for them?

I agree with @bilts it seems this is better suited for the service wrapper. Right now the service wrapper defines a command-line interface for the service, it would make sense for the wrapper to also define an HTTP interface for invoking the service